### DISCUSSION TITLE
BindingEnergy correlation with Temperature

### DISCUSSION THREAD

**Discussion by DamianoBuratto** [Valdes-Tresanco-MS/gmx_MMPBSA#497] [Q&A]
I am currently conducting several simulations of a system involving two interacting proteins at temperatures ranging from 300K to 380 K. To compute the binding free energy of the two proteins, I am utilizing gmx_MMPBSA with linear PB. 
Do you know if I should be able to observe any correlation between the binding free energy and temperature?
Thank you for your time

**Comment by Valdes-Tresanco-MS** [comment]
This is difficult to answer since it probably depends on the system. You will know after do the calculations
